packages/app/package.json

```diff
@@ -41,7 +41,7 @@
     "@mantine/spotlight": "7.9.2",
     "@microsoft/fetch-event-source": "^2.0.1",
     "@tabler/icons-react": "^3.5.0",
-    "@tanstack/react-query": "^5.56.2",
+    "@tanstack/react-query": "^5.90.2",
```

Comment:
Upgraded to pull in the new **experimental** [streamedQuery](https://tanstack.com/query/latest/docs/reference/streamedQuery) function.

If we want to avoid relying on the experimental feature, this could be implemented more verbosely with `useInfiniteQuery`.